aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,8 +1,301 @@
+
+=============  ===============================================================================================================================  
+Task Name	   Description    
+=============  ===============================================================================================================================  
+clean:js	   A task that uses the rimraf Node deletion module to remove unneeded files and directories files.
+clean:css	   A task that uses the rimraf Node deletion module to remove unneeded files and directories files.
+clean	       A task that calls both the ``clean:js`` and ``clean:css`` tasks.
+min:js	       A task that minifies and concatenates *.js* files.
+min:css	       A task that minifies and concatenates *.css* files.
+min	           A task that calls both the ``min:js`` and ``min:css`` tasks.
+=============  =============================================================================================================================== 
+
+Running Default Tasks
+---------------------
+
+If you haven’t already created a new Web app, create a new ASP.NET Web Application project in Visual Studio 2015.
+
+1.	Select **File** > **New** > **Project** from the menu bar. The **New Project** dialog box is displayed.
+
+	.. image:: bundling-and-minification/_static/01-NewProjectDB.png
+	
+2.	Select the **ASP.NET Web Application** template, choose a project name and click **OK**.
+3.	In the **New ASP.NET Project** dialog box select the **Web Application** template from the **ASP.NET 5 Templates** and click **OK**.
+4.	In **Solution Explorer**, right-click *gulpfile.js* and select **Task Runner Explorer**. 
+
+	.. image:: bundling-and-minification/_static/02-SolutionExplorer-TaskRunnerExplorer.png
+	
+	**Task Runner Explorer** shows the list of gulp tasks. In the default ASP.NET 5 Web Application template in Visual Studio 2015 there are six tasks included from *gulpfile.js*.
+
+	.. image:: bundling-and-minification/_static/03-TaskRunnerExplorer.png 
+
+5.	Underneath **Tasks** in **Task Runner Explorer** right-click **clean** and select **Run** from the pop-up menu.
+
+	.. image:: bundling-and-minification/_static/04-TaskRunner-clean.png 
+
+**Task Runner Explorer** will create a new tab named **clean** and execute the related clean task as it is defined in *gulpfile.js*.
+
+6.	Next, right-click the **clean** task, then select **Bindings** > **Before Build**.
+
+ 	.. image:: bundling-and-minification/_static/05-TaskRunner-BeforeBuild.png 
+
+	The **Before Build** binding option will allow the clean task to be automatically run before each time you build your project.
+
+Defining and Running a New Task
+-------------------------------
+
+To define a new gulp task, you must modify *gulpfile.js*.
+ 
+1.	Add the following JavaScript to the end of *gulpfile.js*:
+
+.. code-block:: javascript
+
+	gulp.task("first", function () {
+		console.log('first task! <-----');
+	});
+	
+	This task is named ``first`` and simply displays a string. 
```

Comment:
format error, get rid of space so that doesn't show up as code. check build
This task is named ``first``